<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pluginModbusGUI::ClientControlWidget</class>
 <widget class="QDialog" name="pluginModbusGUI::ClientControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>43</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="connectButton">
     <property name="text">
      <string>&amp;Connect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="disconnectButton">
     <property name="text">
      <string>&amp;Disconnect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pollButton">
     <property name="text">
      <string>&amp;Poll</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
